modular-docs-manual/content/topics/module_anchor-and-file-names-concept.adoc

```diff
@@ -0,0 +1,52 @@
+[#anchor-and-file-names]
```

Comment:
Apparently the discussion between @tradej and me on anchor names and file names was removed when I pushed the updated branch. I thought it would be retained in the previous commit, but I guess not.

At any rate, at the risk of throwing out a bombshell, I've actually re-proposed for anchors and files to actually be named the same. @tradej had rationale for naming anchors the same as the module heading, as we do traditionally (and which I previously said, in contradiction to myself), but I believe that now that each file will or should consist of a single anchor typically, it might make more sense for the file and anchor to match, with the exception of underscores vs. dashes (and that distinction has merit as well, which I can explain). This seems clearer to me when we cross-reference, etc. Previously, before modules, having anchor names that matched module headings made more sense because we had many anchored headings in a single file. But with a single anchor per file, for the most part, it doesn't make much sense to me to have two different naming conventions anymore. 

Bottom line, whatever we decide for the file names, we should probably apply it also to the anchor names for simplicity. Whether we agree with what I'm proposing or not is valuable discussion we need to have, but I do think they should be the same for both, either way. 

I'd like to hear more about @tradej 's reasons for the contrary.